packages/manifest/src/pixi.ts

```diff
@@ -172,13 +174,27 @@ function collect(
                 ) as PixiManifestEntry[];
         }
 
-        result.forEach((entry) =>
+        const hasIgnoreFileExtensions
+            = options.ignoreFileExtensions !== undefined
+            && options.ignoreFileExtensions.length > 0;
+
+        result.forEach((entry, index) =>
         {
-            if (
-                !entry.data?.tags
-                && (Object.keys(tree.fileTags).length > 0
-                    || Object.keys(tree.pathTags).length > 0)
-            )
+            if (hasIgnoreFileExtensions)
+            {
+                if (
+                    options.ignoreFileExtensions?.some((extensionName) =>
```

Comment:
question: do you need the '?' (I assume you do and typescript is complaining)
```suggestion
                    options.ignoreFileExtensions.some((extensionName) =>
```

Reply:
yea it is complaining that it can be `undefined`  :))

Reply:
yeah after chatting with Alex, its because its in a different block to the hasIgnoreFileExtensions check.